src/mumble/GlobalShortcut_win.h

```diff
@@ -19,6 +19,8 @@
 
 #define DIRECTINPUT_VERSION 0x0800
 #include <dinput.h>
+#include <windows.h>
+#include <psapi.h>
```

Comment:
Please move this to `GlobalShortcut_win.cpp`.

Reply:
(It will be `<tlhelp32.h>` with `CreateToolhelp32Snapshot()`).

Reply:
I'm quite confused on what you'd like me to do with this. `#include <psapi.h>` is in the windows header -- do you want me to move that and remove `#include <windows.h>`?

Reply:
Correct.